src/cli/onefuzz/cred_wrapper.py

```diff
@@ -0,0 +1,66 @@
+# Wrap credentials from azure-identity to be compatible with SDK that needs msrestazure or azure.common.credentials
```

Comment:
New files need a copyright notice:
```suggestion
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.
#
# Wrap credentials from azure-identity to be compatible with SDK that needs msrestazure or azure.common.credentials
```

Reply:
Not our code. This is required due to AppInsights module using older auth (and there is no newer AppInsights)

see https://docs.microsoft.com/en-us/azure/developer/python/azure-sdk-authenticate-hosted-applications

![image](https://user-images.githubusercontent.com/7307755/147987301-cb9a34bb-0192-4548-a95a-91a69a22c5ef.png)